lfs/transfer_queue.go

```diff
@@ -55,16 +117,14 @@ type TransferQueue struct {
 	wait          sync.WaitGroup
 	oldApiWorkers int // Number of non-batch API workers to spawn (deprecated)
 	manifest      *transfer.Manifest
-	rmu           sync.Mutex        // rmu guards retryCount
-	retryCount    map[string]uint32 // maps OIDs to number of retry attempts
-	// maxRetries is the maximum number of retries a single object can
-	// attempt to make before it will be dropped.
-	maxRetries uint32
+	rc            *retryCounter
 }
 
 // newTransferQueue builds a TransferQueue, direction and underlying mechanism determined by adapter
 func newTransferQueue(files int, size int64, dryRun bool, dir transfer.Direction) *TransferQueue {
-	logPath, _ := config.Config.Os.Get("GIT_LFS_PROGRESS")
+	cfg := config.Config
```

Comment:
This is a pattern that we have in a few places. Instead of wiring up sending a `*config.Configuration` through all of `newTransferQueue`
's call-sites, leave a tiny reminder that the value is interchangeable.